lua/telescope/config.lua

```diff
@@ -323,6 +323,33 @@ local telescope_defaults = {
                           Default: 1000
     ]],
   },
+  previewers = {
```

Comment:
If it is possible i would want to merge this after docs refactor. I dont have the strength to rebase config.lua again :laughing:

Reply:
Totally understand. I _shouldn't_ have meaningful time to work on this before Saturday again anyways? Otherwise I'm getting to distracted from supposedly more relevant stuff :sweat_smile: